docs/website/generate_extra_files.sh

```diff
@@ -31,6 +31,10 @@ cmake -G Ninja \
   -DIREE_BUILD_DOCS=ON
 cmake --build "${BUILD_DIR}" --target iree-doc
 
+if (( IREE_USE_CCACHE == 1 )); then
+  ccache --show-stats
+fi
```

Comment:
results of cache hits:
https://github.com/openxla/iree/actions/runs/5348982378/jobs/9699572753#step:8:844
```
Cacheable calls:    335 /  351 (95.44%)
  Hits:             334 /  335 (99.70%)
```
5m42s build -> 2m41s build
8m3s total -> 4m9s total